content/en/docs/reference/version-policy.md

```diff
@@ -239,7 +218,7 @@ one of the following Kubeflow SDKs and command-line interfaces
         <td>0.7.1</td>
       </tr>
       <tr>
-        <td><a href="/docs/methods/kfctl/kustomize/">kfctl</a> 
+        <td>kfctl
           (<a href="https://github.com/kubeflow/kfctl">GitHub</a> )
```

Comment:
Kfctl was also depricated in Kubeflow v1.3

Reply:
removed

Reply:
@RFMVasconcelos done